techbuzz/src/main/resources/templates/users/login.html

```diff
@@ -40,6 +40,9 @@ <h3>Login into TechBuzz</h3>
                     </div>
                     <button type="submit" class="btn btn-primary">Login</button>
                 </form>
+                <form id="resendVerificationForm" method="get" th:action="@{/resentVerification}">
+                    <button type="submit" class="btn btn-primary">Resend Verification Link</button>
```

Comment:
Instead of Form submission, it can be a simple <a href="/resentVerification">Resend Verification</a> link.

Reply:
As form 'll help validate the email format and related errors hence used the form. Is there any way we can achieve with just the link i can change that to link.

Reply:
In this form we are not taking email input. It is a simple GET request to show Resent Verification Form.

Reply:
my bad . understood. updated the code